repo-projects/scenario-tests.proj

```diff
@@ -10,12 +10,12 @@
     <!-- Don't use the updated NuGet.config file that includes the live package feeds when testing source-only as in that configuration
          the Microsoft built packages should be used. -->
     <TestArgs Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetSourceOnlyTestOnly)' != 'true'">$(TestArgs) /p:TestRestoreConfigFile=$(OriginalNuGetConfigFile)</TestArgs>
-    <UseBootstrapArcade Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetSourceOnlyTestOnly)' == 'true'">true</UseBootstrapArcade>
+    <UseBootstrapArcade Condition="('$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetSourceOnlyTestOnly)' == 'true') or '$(SkipScenarioTestsDependencies)' == 'true'">true</UseBootstrapArcade>
   </PropertyGroup>
 
   <!-- In source only test mode, do not depend on other repositories as the scenario tests repo needs to be capable of being built
        by itself in a standalone validation job. -->
```

Comment:
Comment should be updated to reflect all conditions.

Reply:
Will do - thanks.

Reply:
Fixed with https://github.com/dotnet/dotnet/pull/327/commits/bd120ef6744a3c4689ba8644ae5a56578b4f5a2d